packages/block-library/src/block/edit.native.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * External dependencies
+ */
+import { Platform, Text, TouchableWithoutFeedback, View } from 'react-native';
+import { partial } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { BottomSheet, Icon, Spinner, Disabled } from '@wordpress/components';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { BlockEditorProvider, BlockList } from '@wordpress/block-editor';
+import { compose, withPreferredColorScheme } from '@wordpress/compose';
+import { parse } from '@wordpress/blocks';
+import { help } from '@wordpress/icons';
+import { requestUnsupportedBlockFallback } from '@wordpress/react-native-bridge';
+
+/**
+ * Internal dependencies
+ */
+import styles from './editor.scss';
+import EditTitle from './edit-title';
+
+class ReusableBlockEdit extends Component {
+	constructor( { reusableBlock } ) {
+		super( ...arguments );
+
+		this.setBlocks = this.setBlocks.bind( this );
+		this.toggleSheet = this.toggleSheet.bind( this );
+		this.requestFallback = this.requestFallback.bind( this );
+
+		if ( reusableBlock ) {
+			// Start in edit mode when we're working with a newly created reusable block
+			this.state = {
+				// Since edition is not supported yet isEditing is always false
+				isEditing: false,
+				blocks: parse( reusableBlock.content ),
+				showHelp: false,
+				sendFallbackMessage: false,
+			};
+		} else {
+			// Start in preview mode when we're working with an existing reusable block
+			this.state = {
+				isEditing: false,
+				blocks: [],
+				showHelp: false,
+				sendFallbackMessage: false,
+			};
+		}
+	}
+
+	componentDidMount() {
+		this.props.fetchReusableBlock();
```

Comment:
Previously this call was conditioned to not having data of the reusable block, I had to remove that because with the edition through UBE the component is re-mounted when is modified and I have to refresh the data doing another fetch.

I saw there's a way to listen to modification trough UBE with [`subscribeReplaceBlock`](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-bridge/index.js#L99-L112) but it only returns the content so if the user changes the title we need to fetch everything again 😢 .